CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Bug Fixes
 ---------
 
 - Fixed a bug in the management of SSRCs in Chrome. (JSDK-2032)
+- Fixed the getStats api deprecation warnings in Firefox. (JSDK-1227)
```

Comment:
Nitpicks:

* We can probably drop "the"
* Please place "getStats" in backticks, e.g. `getStats`.
* Please capitalize "api" as "API".